scripts/lib/custom_networking.sh

```diff
@@ -0,0 +1,65 @@
+func configure_custom_networking(){
```

Comment:
Is this script solely for testing purposes? If yes, consider moving it to amazon-vpc-cni-k8s/test directory

Reply:
Yeah it is only for setting up custom networking, I added it here since we have performance script. I can move it to /test directory.